chain/actions/src/payload.rs

```diff
@@ -530,22 +530,20 @@ pub fn convert_vrf_parameters(
     domain_separator: &Hash,
 ) -> Vrfparameters {
     // skip the secp256k1 curvepoint prefix
-    let v = off_chain.get_decompressed_v().unwrap().to_bytes();
-    let s_b = off_chain
-        .get_s_b_witness(signer, &ticket_hash.into(), domain_separator.as_slice())
-        .to_bytes();
+    let v = off_chain.get_decompressed_v().unwrap().serialize_uncompressed();
```

Comment:
I'm against using the `unwrap` inside the code like this. If the value is always present, then let's pass it directly. If it may not be present, let's pattern match. Unnecessary risk to execution.

Reply:
@robertkiel how do we know the point decompression cannot fail here ?

Reply:
changed, good point